_redirects

```diff
@@ -1 +1,3 @@
 /docs/1.2.x/*                /docs/*
+https://docs.surrealdb.com/docs/*   https://surrealdb.com/docs/surrealdb/:splat 200
+https://docs.surrealdb.com/*        https://surrealdb.com/docs/:splat 200
```

Comment:
We need to actually write the redirect functionality on the Cloudfront viewer-request...

Reply:
As in add a cloudfront site for docs.surrealdb.com? I don't see the issue with dealing with the docs. -> www. redirect by netlify as it's already hosted there anyways...

Reply:
No no this is good. But we need to ensure that all necessary redirects (different to these ones) are written on the www.surrealdb.com side too...